nss-day-cohort-38/bangazon-workforce-management-alpha-orionis
# Add Employee Form

## Description
An employee can now be added from the employee list page.

## Types of changes
form.html and form.py have been added to the employee template and view, respectively.
a model_factory function has been added to the models directory

##To test
```
git fetch --all
git checkout cn-add-employee
```
Test the add employee function to make sure an employee can be added. Once an employee is added, the user should be redirected to the list of employees, including the newest one added.
